.github/workflows/build.yml

```diff
@@ -24,8 +24,8 @@ jobs:
         uses: peaceiris/actions-gh-pages@bbdfb200618d235585ad98e965f4aafc39b4c501
         with:
           # personal token can be generated at https://github.com/settings/tokens,
-          # and added to "Settings"->"Secrets"
-          personal_token: ${{ secrets.PERSONAL_TOKEN }}
```

Comment:
I have a question.

In the [build.yml](https://github.com/seismo-learn/software/blob/main/.github/workflows/build.yml) of software repo,

`github_token: ${{ secrets.GITHUB_TOKEN }}`

Here, previous `personal_token` is named as `secrets.PERSONAL_TOKEN`.

What's the difference between the two names?

Reply:
GITHUB_TOKEN is a token automatically generated for the current github actions. So you don't need to generate it before using. See https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret for more explanations.

However, GITHUB_TOKEN can **NOT** make changes to external repositories. Like here, this workflow is trying to push to HTML files to another repository `seismo-learn.github.io`, so I have to use a "personal access token". 

The personal access token is generated by me (you can generate your own tokens at https://github.com/settings/tokens), and I added it to this repository at https://github.com/seismo-learn/website/settings/secrets/actions.

As explained in the top comment, github allows repository (local) tokens and organization (global) tokens. I feel that it's more convenient to share the tokens within the organization, so that we don't have to generate personal access tokens for each repository.